### PR TITLE
Adding parenthesis

### PR DESCRIPTION
**Pull request by ShimonOhayon**
##### Summary
 I came across these small typo, Fixing for good measure  
##### Component Name
Installation script
##### Test Plan
All Build\CI tests should pass

##### Additional Information
Not Urgent

### CODE REVIEW

File: packaging/installer/README.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ title: "Installation guide"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/installer/README.md
 -->
 
-import { Install, InstallBox } from '../../../src/components/Install/'
```
Comment (ilyam8) [inline packaging/installer/README.md:6]: That is probably used by https://learn.netdata.cloud/, but not sure 🤷‍♂️ 

Why do you think we can delete it?
---

Learn components: https://github.com/netdata/learn/tree/master/src/components

Other such imports:

```cmd
[ilyam@pc netdata]$ grep "import {" --include \*.md --include \*.mdx -R *
docs/get-started.mdx:import { OneLineInstall } from '../src/components/OneLineInstall/'
docs/get-started.mdx:import { Install, InstallBox } from '../src/components/Install/'
docs/store/change-metrics-storage.md:import { Calculator } from '../../src/components/agent/dbCalc/'
packaging/installer/README.md:import { Install, InstallBox } from '../../../src/components/Install/'
```

Comment (ShimonOhayon) [reply packaging/installer/README.md:6]: I wasn't aware of it, You probably right.

Comment (ilyam8) [reply packaging/installer/README.md:6]: Ok, i guess that InstallBox thing is used to create https://learn.netdata.cloud/docs/agent/packaging/installer#alternative-methods

Comment (ShimonOhayon) [reply packaging/installer/README.md]: I see, Reverted. thank you

**Review by iigorkarpov — APPROVED** [review]

**Review by ilyam8 — CHANGES_REQUESTED** [review]
Before merging, the following is required:
 - generate kickstart.sh md5 checksum.
 - update the checksum in the [kickstart.md](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/kickstart.md#verify-script-integrity).

After merging:
 - @Ferroin uploads new kickstart.sh to my-netdata.io.

**Comment by ShimonOhayon** [comment]
> Before merging, the following is required:
> 
> * generate kickstart.sh md5 checksum.
> * update the checksum in the [kickstart.md](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/kickstart.md#verify-script-integrity).
> 
@ilyam8 , Done
```
 ~/forked/netdata | fix_small_typos  md5sum packaging/installer/kickstart.sh                                                                                                                                                           255
7e8b449ca44e49b7074b5b9d6022cbcc  packaging/installer/kickstart.sh

```

**Review by ilyam8 — APPROVED** [review]

**Comment by ilyam8** [comment]
@ShimonOhayon thanks! It is good now. Let @Ferroin merge the PR, he needs to upload the file.

**Review by iigorkarpov — APPROVED** [review]

**Review by maneamarius — APPROVED** [review]

**Comment by Ferroin** [comment]
@ShimonOhayon For future reference, when a PR needs to be updated to the latest master we usually prefer to rebase than back-merge master into the PR branch, as it keeps the history cleaner on the branch which in turn makes it easier to review.

**Pull request merged**